examples/kubernetes/delete_ceph_cluster.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 
 kubectl delete namespace ceph
+kubectl label nodes --all node-type-
```

Comment:
Haha yes it looks funny! This tells kubernetes to remove the `node-type` label from all nodes. The remove is indicated by the trailing `-`.